tigerpath/majors_and_certificates/majors/SLA.json

```diff
@@ -11,13 +11,13 @@
   "contacts": [
     {
       "type": "Chair",
-      "name": "Michael A. Wachtel",
-      "email": "wachtel@princeton.edu"
+      "name": "Ilya Vinitsky",
+      "email": "vinitsky@princeton.edu"
     },
     {
       "type": "Director of Undergraduate Studies",
-      "name": "Elena Fratto",
-      "email": "efratto@princeton.edu"
+      "name": "Katherine M.H. Reischl",

```

Comment:
Looks like this has changed already. It's Ellen B. Chances now: https://ua.princeton.edu/academic-units/department-slavic-languages-and-literatures